axelor-base/src/main/java/com/axelor/apps/base/web/LanguageController.java

```diff
@@ -0,0 +1,12 @@
+package com.axelor.apps.base.web;
+
+import com.axelor.apps.base.db.Language;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+
+public class LanguageController {
+  public void convertToLowercase(ActionRequest request, ActionResponse response) {
+    Language language = request.getContext().asType(Language.class);
+    response.setValue("code", language.getCode().toLowerCase());
```

Comment:
Check if language.getCode() is not null or empty (you can use Strings.isNullOrEmpty) before doing that, because when I delete the code on a language, then I get an NPE : 
``` java
2023-11-22 16:06:03.764 ERROR 55785 --- [nio-8080-exec-5] c.a.a.b.s.exception.TraceBackService     : java.lang.NullPointerException
	at com.axelor.apps.base.web.LanguageController.convertToLowercase(LanguageController.java:10)
```